rotkehlchen/tests/exchanges/test_kucoin.py

```diff
@@ -727,24 +695,39 @@ def test_query_asset_movements_sandbox(
             fee=Fee(FVal('0.0001')),
             link='',
         ),
+        AssetMovement(
+            location=Location.KUCOIN,
+            category=AssetMovementCategory.DEPOSIT,
+            timestamp=Timestamp(1612556651),
+            address='0x5f047b29041bcfdbf0e4478cdfa753a336ba6989',
+            transaction_id='5bbb57386d99522d9f954c5b',
+            asset=Asset('LINK'),
+            amount=AssetAmount(FVal('1000')),
+            fee_asset=Asset('LINK'),
+            fee=Fee(FVal('0.01')),
+            link='',
+        ),
         AssetMovement(
             location=Location.KUCOIN,
             category=AssetMovementCategory.WITHDRAWAL,
-            timestamp=Timestamp(1612556765),
+            timestamp=Timestamp(1612556652),
             address='1DrT5xUaJ3CBZPDeFR2qdjppM6dzs4rsMt',
-            transaction_id='b893c3ece1b8d7cacb49a39ddd759cf407817f6902f566c443ba16614874ada5',
+            transaction_id='b893c3ece1b8d7cacb49a39ddd759cf407817f6902f566c443ba16614874ada4',
             asset=Asset('BSV'),
             amount=AssetAmount(FVal('2.5')),
             fee_asset=Asset('BSV'),
             fee=Fee(FVal('0.25')),
-            link='5c2dc64e03aa675aa263f1a5',
+            link='5c2dc64e03aa675aa263f1a4',
         ),
     ]
 
     def get_endpoints_response():
         results = [
-            f'{deposits_response}',
-            f'{withdrawals_response}',
+            f'{deposits_response_1}',
+            f'{deposits_response_2}',
+            f'{withdrawals_response_1}',
+            f'{withdrawals_response_2}',
+            f'{withdrawals_response_3}',  # never requested as
```

Comment:
what do you mean never requested as?

Reply:
The `as` is a typo, "is never requested" or I can just remove the comment.
If pagination works as expected (breaking the requesting loop), this withdrawal won't be processed.

Reply:
perhaps just modify the comment explaining what you just said here :)